public/index.html

```diff
@@ -26,5 +41,8 @@
 		<p>Compare law as it is now and as it could be.</p>
 
 		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<div id="footer">
+		<a href="./tracking.html">tracking<a>
```

Comment:
- Missing indentation.
- Use proper capitalisation.
- You probably want to use a list.
- Don't add `./` in relative URLs, they are redundant.
- Use a more understandable name such as `Cookies` or `Analytics policy`.